### DISCUSSION TITLE
Difficulty installing organ sample set: ".orgue" vs ".organ"

### DISCUSSION THREAD

**Discussion by jwallbillich** [GrandOrgue/grandorgue#1218] [Help]
Hello, 

I followed the instructions in Lard Palo's video: https://www.youtube.com/watch?v=JoF1DxtoS3g&t=274s but when I go to load an organ the program is looking for the ".orgue" file extension so it does not see the ".organ." I have successfully downloaded the files for the sample organ and unzipped them. I have located the ".organ" in my file manager as he has done in his video, but when I try to do so in the program I hit this roadblock. 

I am running pop_os, although I am not very knowledgeable about linux. I think it functions similarly to ubuntu. I imagine this has something to do with assigning the directories, right? This is new to me. 

I have seen a few discussions on the topic, but I must ask for help. I am incompetent and would greatly appreciate a step by step explanation. Thank you in advance! and Thank you to the creators of this wonderful program. I am enjoying using the built in sample thus far!

**Comment by oleg68** [comment]
Use `File`->`Open` instead of `Install` for the .organ file
